### ISSUE TITLE
Pydantic V1 validated AuthenticationCredential & RegistrationCredential do not pass verify

### ISSUE THREAD

**Issue by bschoenmaeckers** [duo-labs/py_webauthn#185]
When verifying authentication or registration responses that are are "validated" pydantic models you will get an error that `id and raw_id were not equivalent`.  This is because on pydantic V1 base64 encoded bytes are not converted to bytes correctly when calling `validate()` on the model.

This is especially a problem when using this library with fastapi because it uses `validate()` internally to validate request data.
 ## Example

```python
app = fastapi()

@app.post("/passkey/verify")
def passkey_verify(authentication_response: AuthenticationCredential) -> AuthLoginResponse:
        # further logic omitted...

        # This will throw InvalidAuthenticationResponse: id and raw_id were not equivalent
        authentication = verify_authentication_response(
            credential=authentication_response,
            ...
        )
```

**Comment by MasterKale** [comment]
Thanks for reporting this @bschoenmaeckers, what a strange issue. I suppose it only highlights how infrequently I ever call `.validate()` on a Pydantic model...

I'll review #186 shortly.

**Comment by jwag956** [comment]
It also appears that this incorporates my fix for user_handle - so possibly only this one is required?

**Comment by MasterKale** [comment]
If I proceed with #195 then I'll end up closing this out as the project won't use Pydantic anymore 😬

**Comment by MasterKale** [comment]
I've merged #195 which removed Pydantic as a dependency of this project so I'm closing out any PR's and issues related to Pydantic as they're no longer needed.
